scripts/exrap_dump.sh

```diff
@@ -68,9 +69,14 @@ set +u
 # ------------------------------------------------------------------------
 # Dump group #1 (non-pb) = 1bamua 1bmhs esamua esmhs atms mtiasi sevcsr
 #                          gpsro esiasi iasidb esatms atmsdb sevasr amsr2
+<<<<<<< HEAD
```

Comment:
Need to remove or reconcile these merge conflict notations

Reply:
I resolved this conflict in feature/dev.iss49636.TAC2BUFR_upper-air.v1.1